subprojects/docs/src/docs/userguide/playPlugin.xml

```diff
@@ -287,10 +287,10 @@
                     <literal>compilePlayBinaryPlayTwirlTemplates</literal>
                 </td>
                 <td>
-                    Twirl HTML templates
+                    Twirl templates
                 </td>
                 <td><apilink class="org.gradle.play.tasks.TwirlCompile"/></td>
-                <td>Compiles HTML templates with the Twirl compiler.</td>
+                <td>Compiles templates with the Twirl compiler.</td>
```

Comment:
We should add:
- A list of supported built-in template types
- A short section describing how to define a custom template
I'd be OK if most of the description was in the DSL ref / Javadoc, and linked in the manual.